src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -171,8 +175,23 @@ export default function Overlay() {
           awakingRef.current = true;
           wake(commandTimeout);
         }
+
+        // Ignore chat popup preference, the refresh should always go through and show if called
+        if (command === "refresh" && isPrivileged) {
+          refreshAmbassadors().then(() => {
+            setActiveAmbassador({});
+            setVisibleOption("welcome");
+            const ambassadorLists = document.querySelectorAll(".list-fade");
+            ambassadorLists.forEach((list) => {
+              list.scrollTo({
+                top: 0,
+                behavior: "smooth",
+              });
+            });
+          });
```

Comment:
I don't think we need any of this? A refresh should just silently happen in the background, definitely don't want to forcibly show any popups as a result of it. The only thing we might want is a separate useEffect that watches for whenever `ambassadors` changes and resets `activeAmbassador` if it no longer exists?